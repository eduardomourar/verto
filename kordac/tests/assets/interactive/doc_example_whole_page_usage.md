{interactive name="binary-cards" type="in-page" parameters="digits=5&start=BBBBB" text="Binary Cards Interactive"}
