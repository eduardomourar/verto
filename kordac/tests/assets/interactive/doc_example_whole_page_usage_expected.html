<a class="btn btn-interactive-link" href="{% url 'interactive' name='binary-cards' %}?digits=5&start=BBBBB">
 <img class="btn-interactive-link-image" src="{% static 'binary-cards/thumbnail.png' %}">
 <div class="btn-interactive-whole-page-text">
  Binary Cards Interactive
 </div>
</a>
